### PR TITLE
Update concept.md

### PR DESCRIPTION
**Pull request by Walkman100**

### CODE REVIEW

File: concept.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Concept for Spencer
 
 <h2>What is it?</h2>
 
-A front-end hybridzed programming language with Bash.
+A front-end hybrized programming language with Bash.
```
Comment (Walkman100) [inline concept.md]: I didn't change this

**Comment by Walkman100** [comment]
oi m8

**Comment by Walkman100** [comment]
I wasn't done

**Comment by deavmi** [comment]
lol soz :smile:

**Pull request merged**